RandomClb panic when no SRV records found

RandomClb.GetAddress() panics when there are no SRV records found for the given name. RoundRobinClb guards against this, returning fmt.Errorf("no SRV records found"), but RandomClb gives:

```
panic: invalid argument to Intn

goroutine 16 [running]:
runtime.panic(0x5cf040, 0xc208000710)
        /usr/local/go/src/pkg/runtime/panic.c:279 +0xf5
math/rand.(*Rand).Intn(0xc208000160, 0x0, 0x0)
        /usr/local/go/src/pkg/math/rand/rand.go:95 +0x71
math/rand.Intn(0x0, 0x698577)
        /usr/local/go/src/pkg/math/rand/rand.go:195 +0x34
github.com/benschw/dns-clb-go/randomclb.(*RandomClb).GetAddress(0xc2080005a0, 0x698577, 0x17, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/sheldonh/go/src/github.com/benschw/dns-clb-go/randomclb/random.go:27 +0x126
main.main()
        /home/sheldonh/go/src/github.com/sheldonh/foo/foo.go:24 +0x36d
```

--- a/randomclb/random.go
+++ b/randomclb/random.go
@@ -1,6 +1,7 @@
 package randomclb
 
 import (
+	"fmt"
 	"github.com/benschw/dns-clb-go/dns"
 	"math/rand"
 )
@@ -22,8 +23,11 @@ func (lb *RandomClb) GetAddress(name string) (dns.Address, error) {
 	if err != nil {
 		return add, err
 	}
-	//	log.Printf("%+v", srvs)
+	if len(srvs) == 0 {
+		return add, fmt.Errorf("no SRV records found")
+	}
 
+	//	log.Printf("%+v", srvs)
 	srv := srvs[rand.Intn(len(srvs))]
 
 	ip, err := lb.dnsLib.LookupA(srv.Target)

